src/modules/claims/claims.service.ts

```diff
@@ -663,28 +667,28 @@ export class ClaimsService {
     };
 
     if (registrationTypes.includes(RegistrationTypes.OffChain)) {
+      const vp = await this.issueVerifiablePresentation({
```

Comment:
This code is for this task. The other changes in this file were changes needed after updating ew-credentials version

Reply:
Can you rebase against `develop`  please? I think https://github.com/energywebfoundation/iam-client-lib/pull/615 updated ew-credential as well (to an even higher version).
Then I think this PR can be isolate to just "add credentialStatus to PublicClaim"

Reply:
just rebased this! thanks